locationsharinglib/locationsharinglib.py

```diff
@@ -274,7 +274,26 @@ def _get_data(self):
         self._logger.debug(response.text)
         try:
             output = json.loads(response.text.split("'", 1)[1])
-            people = [Person(info) for info in output[0]]
+            people = []
+            if output[0] is not None:
+                people.extend([Person(info) for info in output[0]])
+
+            if output[9] is not None and output[9][1] is not None:
```

Comment:
Same here.

Reply:
Same as my previous comment with regards to output[9]. But I agree that the check for output[9][1] is probably unnecessary.